Common/src/wall_model.cpp

```diff
@@ -306,9 +306,11 @@ void CWallModel1DEQ::WallShearStressAndHeatFlux(const su2double tExchange, const
     if (y_cv[0] * sqrt(tauWall / rho) / (mu_lam / rho) > 1.0)
       SU2_MPI::Error("Y+ greater than one: Increase the number of points or growth ratio.", CURRENT_FUNCTION);
 
-    /* Define a norm
-     */
-    if (abs(1.0 - tauWall / tauWall_prev) < tol && abs(1.0 - qWall / qWall_prev) < tol) {
+    /* Define a norm */
+    bool tau_converged = abs(1.0 - tauWall / fmax(tauWall_prev, 1e-16)) < tol;
+    bool q_converged = abs(1.0 - qWall / fmax(qWall_prev, 1e-16)) < tol;
```

Comment:
We have a constant defined for this
```suggestion
    bool tau_converged = abs(1.0 - tauWall / fmax(tauWall_prev, EPS)) < tol;
    bool q_converged = abs(1.0 - qWall / fmax(qWall_prev, EPS)) < tol;
```